spec/lib/video_info_spec.rb

```diff
@@ -24,6 +24,29 @@
       expect { VideoInfo.new(url, options) }.to raise_error(VideoInfo::UrlError)
     end
   end
+  
+  describe ".disable_providers" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.